poetry.lock

```diff
@@ -1,10 +1,16 @@
+# This file is automatically @generated by Poetry and should not be changed by hand.
```

Comment:
Out of curiosity, which poetry version are you using locally @varunshankar ? Haven't seen this comment before, neither the parameter `files` which now shows up for every package.

Reply:
Got this info from `poetry about`

Poetry - Package Management for Python

Version: 1.3.1
Poetry-Core Version: 1.4.0

Reply:
Oh I see, can you please downgrade to `1.2.2`, that's the version we use in the CI and for building Docker images at the moment.

Reply:
Ya, I have downgraded poetry now to `1.2.2` and created a new lock file